normandy/classifier/api.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers, views, status
+from rest_framework.response import Response
+
+from normandy.bundler.serializers import BundleSerializer
+from normandy.classifier.models import Client
+from normandy.bundler.models import Bundle
+
+
+class FetchBundle(views.APIView):
+
+    serializer_class = BundleSerializer
+
+    class Parameters(serializers.Serializer):
```

Comment:
what is this madness its not in the docs

where did you learn such things

and where can i learn too